.github/workflows/nextest.yml

```diff
@@ -73,17 +73,6 @@ jobs:
       - name: Install Vyper
         run: pip install vyper~=0.4.0
 
-      - name: Forge RPC cache
```

Comment:
Are this changes from upstream tincho or is this ours? If they are, do we have to remove it?

Reply:
This was a step @nbaztec removed in yesterday's [PR](https://github.com/matter-labs/foundry-zksync/pull/869), together with the Python downgrade.
So, I'm redirecting the question now to him.

Reply:
I don't think I committed these changes, and if I did it wasn't intentional. We should have them the same as upstream.

Reply:
cool, thanks!